Xamarin.Forms.Platform.Android/IButtonLayoutRenderer.cs

```diff
@@ -1,12 +1,9 @@
-﻿using System;
-using Android.Support.V7.Widget;
+﻿using Android.Support.V7.Widget;
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public interface IButtonLayoutRenderer
+	public interface IButtonLayoutRenderer : IVisualElementRenderer
 	{
-		AppCompatButton View { get; }
-		Button Element { get; }
-		event EventHandler<VisualElementChangedEventArgs> ElementChanged;
+		new AppCompatButton View { get; }
```

Comment:
@mattleibow @samhouts I had to add this back because the IVisualElementRenderer.View is just going to return the renderer itself for renderers that aren't fast renderers. So all Text was showing up blank in the Control Gallery